cmd/aws-janitor-boskos/main.go

```diff
@@ -60,6 +61,7 @@ var (
 	skipRoute53ManagementCheck = flag.Bool("skip-route53-management-check", false, "If true, skip managed zone check and managed resource name check.")
 	enableDNSZoneClean         = flag.Bool("enable-dns-zone-clean", false, "If true, clean DNS zones.")
 	enableS3BucketsClean       = flag.Bool("enable-s3-buckets-clean", false, "If true, clean S3 buckets.")
+	skipResourceRecordSetTypes = flag.String("skip-resource-record-set-types", "NS,SOA", "A list of resource record types which should not be deleted, splitted using comma.")
```

Comment:
We set the default value to be "NS,SOA" here, maybe we should leave it empty since it influences all users?

Reply:
`--skip-route53-management-check=true` would not work without it, so setting by default is preferred, I think. It would not do anything with default `--skip-route53-management-check=false` anyway.